src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -297,7 +333,7 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
       try {
         let facilityData;
         if (featureVAOSServiceVAAppointments) {
-          facilityData = getAdditionalFacilityInfoV2(data[0]);
+          facilityData = getAdditionalFacilityInfoV2(data.flat());
```

Comment:
The 'data' array now contains upcoming and pending appointments so get facility data for both appointment types.